src/components/ImageView/index.tsx

```diff
@@ -78,32 +67,23 @@ function ImageView({isAuthTokenRequired, url, fileName, onError}) {
         setIsZoomed(false);
     };
 
-    const imageLoad = ({nativeEvent}) => {
+    const imageLoad = ({nativeEvent}: {nativeEvent: ImageViewTypes.ImageLoadNativeEventData}) => {
```

Comment:
Let's use `NativeSyntheticEvent<ImageLoadNativeEventData>` on the event variable